pints/_log_likelihoods.py

```diff
@@ -117,6 +117,137 @@ def __call__(self, x):
                       - np.sum(autocorr_error**2, axis=0) / (2 * sigma**2))
 
 
+class GaussianIntegratedUniformLogLikelihood(pints.ProblemLogLikelihood):
+    r"""
+    Calculates a log-likelihood assuming independent Gaussian-distributed noise
+    at each time point where it is assumed that the dependence on
+    :math:`\sigma\sim U(a,b)` has been integrated out of the joint posterior,
+
+    .. math::
+        \begin{align} p(\theta|X) &= \int_{0}^{\infty} p(\theta, \sigma|X)
+        \mathrm{d}\sigma\\
+        &\propto \int_{0}^{\infty} p(X|\theta, \sigma) p(\theta, \sigma)
+        \mathrm{d}\sigma,\end{align}
+
+    A possible advantage of this likelihood compared with using a
```

Comment:
This sentence is a bit broken!

Reply:
@MichaelClerx Yep, fair cop on the previous two messages. Have now cleaned up the language a bit.